addon/components/frost-select-dropdown.js

```diff
@@ -91,7 +91,7 @@ export default Component.extend({
     ]
       .join(';')
 
-    return Ember.String.htmlSafe(style)
+    return EmberString.htmlSafe(style)
```

Comment:
@juwara0 one advantage is if you make a typo here:
```js
import Ember from 'ember'
const {Component, Strung: EmberString} = Ember
import computed, {readOnly} from 'ember-computed-decorators'

export default Component.extend({
  @readOnly
  @computed('title')
  header (title) {
    return EmberString.htmlSafe(`<h1>${title}</h1>`)
  }
})
```
it will be caught at compile time whereas the following typo won't:
```js
import Ember from 'ember'
const {Component} = Ember
import computed, {readOnly} from 'ember-computed-decorators'

export default Component.extend({
  @readOnly
  @computed('title')
  header (title) {
    return Ember.Strung.htmlSafe(`<h1>${title}</h1>`)
  }
})
```